lang/cs/Org.Apache.REEF.Common/Files/REEFFileNames.cs

```diff
@@ -338,12 +339,21 @@ public string GetPidFileName()
         }
 
         /// <summary>
+        /// The filename of the file that contains the driver name server address and port.
+        /// </summary>
+        /// <returns>File name that contains the dfs path for the DriverNameServerEndpoint</returns>
+        public string GetDriverJavaBridgeEndpoint()
```

Comment:
The file contains endpoint. But the method still returns file name. Name it as GetDriverJavaBridgeEndpointFile() ?